src/Core.UnitTests/Hotspots/HotspotAnalysisConfigurationTests.cs

```diff
@@ -1,82 +0,0 @@
-﻿/*
```

Comment:
Git diff didn't pick up the fact that I renamed this file and added some new stuff, rather than delete and create